middleware/dedupe.js

```diff
@@ -65,6 +65,14 @@ function dedupeResults(req, res, next) {
 function isPreferred(existing, candidateReplacement) {
   // NOTE: we are assuming here that the layer for both records is the same
 
+  var isOA = _.flow(_.property('source'), _.eq.bind(null, 'openaddresses'));
```

Comment:
We ❤️ and use lodash here quite a bit, but this is a master class in its application!! 🙇🏻🙇🏻‍♀️